xbmc/interfaces/json-rpc/SystemOperations.cpp

```diff
@@ -95,6 +96,21 @@ JSONRPC_STATUS CSystemOperations::Reboot(const std::string &method, ITransportLa
     return FailedToExecute;
 }
 
+JSONRPC_STATUS CSystemOperations::InhibitIdleShutdown(const std::string &method, ITransportLayer *transport, IClient *client, const CVariant &parameterObject, CVariant &result)
+{
+  if ((parameterObject["inhibit"].isString() &&
+       parameterObject["inhibit"].asString().compare("toggle") == 0) ||
+      (parameterObject["inhibit"].isBoolean() &&
+       parameterObject["inhibit"].asBoolean() != g_application.IsIdleShutdownInhibited()))
+  {
```

Comment:
with this multi lines if, the curly braces can stay here  for better readability?